example_test.go

```diff
@@ -71,72 +75,131 @@ func TestExample(t *testing.T) {
 		q.Release()
 	}
 
-	// Query the database, storing results in a []Person (wrapped in []interface{})
+	// Query the database, storing results in a []Person (wrapped in []interface{}).
 	{
-		people := []Person{}
-		if err := gocqlx.Select(&people, session.Query("SELECT * FROM person")); err != nil {
+		var people []Person
+		if err := gocqlx.Select(&people, session.Query("SELECT * FROM gocqlx_test.person")); err != nil {
 			t.Fatal("select:", err)
 		}
+		t.Log(people)
 
-		fmt.Printf("%#v\n%#v\n", people[0], people[1])
-		// gocqlx_test.Person{FirstName:"John", LastName:"Doe", Email:[]string{"johndoeDNE@gmail.net"}}
-		// gocqlx_test.Person{FirstName:"Jason", LastName:"Moiron", Email:[]string{"jmoiron@jmoiron.net"}}
+		// [{John Doe [johndoeDNE@gmail.net]} {Jason Moiron [jmoiron@jmoiron.net]}]
```

Comment:
IMO asserts would be less readable. `example_test.go` originated as https://github.com/jmoiron/sqlx#usage rewrite and the idea of commented output it taken from there.